api/Document.json

```diff
@@ -9737,17 +9739,18 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/timeline",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": false,
+              "notes": "Currently Chrome Canary only"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": false,
+              "notes": "Currently Chrome Canary only"
             },
             "edge": {
               "version_added": false
             },
             "firefox": {
-              "version_added": false,
-              "notes": "Currently only supported in Firefox Nightly (version 48)."
+              "version_added": "75"
             },
             "firefox_android": {
               "version_added": false,
```

Comment:
I'm not sure sure about this. I was to understand this was enabled by default.

Reply:
Sorry unclear comment, I meant FF for Android

Reply:
I'm still not 100% sure what you mean. do you mean, add something like the following to the FxA data, here and for `Animation.timeline` ?

```
"flags": [
    {
      "type": "preference",
      "name": "dom.animations-api.timelines.enabled",
      "value_to_set": "true"
    }
  ]
```

Reply:
OK, makes sense, thanks! I've updated the data as appropriatee. I've left off the note about when it changed form read-only to read-write, because there is no way I can find out when this change occurred, and it doesn't really matter now anyway.